HTML Project/Product-landing-page/index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product Landing Page</title>
+    <style>
+      * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+
+      body {
+        font-family: Arial, sans-serif;
+        text-align: center;
+      }
+
+      #nav-bar {
+        position: fixed;
+        top: 0;
+        width: 100%;
+        background: #333;
+        padding: 10px 0;
+        display: flex;
+        justify-content: center;
+        gap: 20px;
+      }
+
+      #nav-bar a {
+        color: white;
+        text-decoration: none;
+        font-size: 18px;
+      }
+
+      #header {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        padding: 20px;
+        background: lightgray;
+        position: relative;
+        margin-top: 50px;
+      }
+
+      #header-img {
+        width: 100px;
+      }
+
+      #video {
+        width: 560px;
+        height: 315px;
+        margin: 20px auto;
+      }
+
+      #form {
+        margin: 20px;
+      }
+
+      #email {
+        width: 300px;
+        padding: 10px;
+        margin: 10px;
+        border: 1px solid #ccc;
+      }
+
+      #submit {
+        background: #28a745;
+        color: white;
+        border: none;
+        padding: 10px 20px;
```

Comment:
Not going to request a change, but using pixels is considered fragile.  You should look at doing `rems` instead of pixels for accessibility purposes